### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by makelove** [Murphy3220/Razor-AHRS-SEN-14001#2]
I had install Sparkfun Razor IMU m0 library for Arduino,then I download your ino files,installed to IMU m0.

but it doesn't working,just red led is on,
nothing in the console.

can you write a detail tutorial about this repo ?
how to do ?

**Comment by Murphy3220** [comment]
Make sure you have it set to SerialUSB in the main Razor AHRS ino file. What OS are you using?  There have been known issues with Linux (see other issue) but I have had it work flawlessly in Windows and Mac.  I just uploaded the code to my IMU and it works as it should.

I will try to write a tutorial when I have time.  You can use the original Razor AHRS tutorial since all of the commands are the same.

**Comment by makelove** [comment]
Both macOS and Ubuntu 16.04. 
I follow the guide of sparkfun
https://learn.sparkfun.com/tutorials/9dof-razor-imu-m0-hookup-guide
success uploaded your code to  IMU m0,but it doesn't working,just red led is on,
nothing in the console.

just this output,I have No idea what is going on.
btw,I have return the IMU to shop,so don't botter if your are busy.
Thanks

**Comment by Murphy3220** [comment]
Yeah I'm not sure what is up with the output not working for Mac/Linux.  I keep reading that it might have something to do with the auto-reset of the Arduino board and/or something with the read/write permissions.  Works flawlessly on Windows though. 

I might have time to work on this again soon and hopefully I can figure out the issue.
